src/Homework/Week3/Pb2/Customer.java

```diff
@@ -0,0 +1,58 @@
+package Homework.Week3.Pb2;
+
+import java.util.List;
+
+public class Customer {
+    private String name;
+    private MemberShip myMemberShip;
+    {
+        myMemberShip = MemberShip.NONE;
+    }
+    private Basket myBasket;
+
+    Customer(String name, MemberShip memberShip)
+    {
+        myMemberShip = memberShip;
+        myBasket = new Basket();
+    }
+
+    void addToBasket(ItemForSale item) {
+        if(item == null) {
+            System.out.println("The item added to the basket is null");
+            return;
+        }
+        myBasket.addItem(item);
+    }
+
+    float basketPrice()
```

Comment:
sa ai grija putin la denumirea metodei, ar trebui sa contina un verb care sa sugereze ce anume se intampla in metoda, de ex: calculateBasketPrice/getBasketPrice